repair_perf_test.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import random
+import datetime
+
+from sdcm.tester import ClusterTester
+from sdcm.db_stats import PrometheusDBStats
+
+
+class RepairPerfTest(ClusterTester):
+    """
+    Test performance of Repair related operations.
+    """
+
+    def test_bootstrap(self):
+        """
+        Init a basic cluster with 3 db nodes, and add a new node
+        """
+        stress_cmd = self.params.get('prepare_write_cmd')
+        stress = self.run_stress_thread(stress_cmd=stress_cmd)
+        self.verify_stress_thread(cs_thread_pool=stress)
+
+        stress_cmd = self.params.get('stress_cmd')
+        stress = self.run_stress_thread(stress_cmd=stress_cmd)
+
+        self.log.info("Adding new node to cluster...")
+        start = datetime.datetime.now()
+        new_node = self.db_cluster.add_nodes(count=1, enable_auto_bootstrap=True)[0]
+        self.monitors.reconfigure_scylla_monitoring()
+        self.db_cluster.wait_for_init(node_list=[new_node], timeout=10800)
+        self.db_cluster.wait_for_nodes_up_and_normal(nodes=[new_node])
+        bootstrap_end = datetime.datetime.now()
+
+        self.verify_stress_thread(cs_thread_pool=stress)
+        read_end = datetime.datetime.now()
+        bootstrap_duration = (bootstrap_end - start).total_seconds()
+        read_workload_time = (read_end - start).total_seconds()
```

Comment:
This test is pretty immature.
It doesn't store results in elastic nor compare to previous results or check for regressions.
It just prints the info to the log.

For now it may be acceptable, but we will need to enhance it later.